gradle.properties

```diff
@@ -10,10 +10,10 @@ sizeofVersion = 0.3.0
 jaxbVersion = 2.3.1
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.6.0
-terracottaApisVersion = 1.6.0
-terracottaCoreVersion = 5.6.0
-terracottaPassthroughTestingVersion = 1.6.0
+terracottaPlatformVersion = 5.7.0-pre1
+terracottaApisVersion = 1.6.0-pre3
+terracottaCoreVersion = 5.6.0-pre13
```

Comment:
I believe the version of galvan associated as a transient dependency with this tc-core release is flawed.  I would allow some time to update all dependencies before publishing this.